CONTRIBUTING.rst

```diff
@@ -0,0 +1,215 @@
+Contribution guide
+==================
+
+Setting up the environment
+--------------------------
+
+1. Install `poetry <https://python-poetry.org/>`_
+2. Run ``poetry install`` to create a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_ and install
+   the dependencies with
+3. If you're working on the documentation and need to build it locally, install the extra dependencies with ``poetry install --with docs``
+4. Install `pre-commit <https://pre-commit.com/>`_
+5. Run ``pre-commit install`` to install pre-commit hooks
+
+
+Code contributions
+------------------
+
+Workflow
+++++++++
+
+1. `Fork <https://github.com/starlite-api/starlite/fork>`_ the `Starlite repository <https://github.com/starlite-api/starlite>`_
+2. Clone your fork locally with git
+3. `Set up the environment <Setting up the environment>`_
```

Comment:
I am not sure if its just inside of the PR, or the blob it loads from for me, but this link 404s

Reply:
Which one?

Reply:
Also I think its better to leave the original hyperlink instead of an alias. The general website will properly hyperlink the phrase any way.

This is what I mean: 
```
`An example RST link <https://www.example.com>`_
```
Alias is best reserved for the API reference instead.

Reply:
hmm.. that's annoying.. I meant line 23 sorry.

Reply:
> Also I think its better to leave the original hyperlink instead of an alias

The name of the hyperlink doesn't make sense grammatically though. This is a list of instructions, therefore phrase in imperative mood. "Setting up the environment" is not imperative mood.